File: proga6/BigLetters.cs
using System;
using RyadkiLibrary;

namespace BigLettersLibrary
{
    public class BigLetters:Ryadki
    {
        public BigLetters(string line) : base(line)
        { }
        public override int RyadokLength()
        {
            return str.Length;
        }
        override public void InsertSymbols()
        {
            string newstr = "";
            for (int i = 1; i < str.Length; i += 2)
            {
                newstr = str.Insert(i, "/");
                str = newstr;
            }
        }
    }
}

File: proga6C/BigLetters.cpp
#pragma once
#include "Header.h"
#include "Ryadki.cpp"

class BigLetters :public Ryadki
{
public:
    BigLetters(string line) :Ryadki(line)
    { }
    int RyadokLength() override
    {
        return str.length();
    }
    void InsertSymbols() override
    {
        string newstr = "";
        for (int i = 1; i < str.length(); i += 2)
        {
            newstr = str.insert(i, "/");
            str = newstr;
        }
    }
};